pyspecProcScripts/load_data.py

```diff
@@ -838,6 +852,7 @@ def hack_field_sweep_v4(s, fl=None):
     "proc_Hahn_echoph": proc_Hahn_echoph,
     "spincore_FID_nutation_v1": proc_FID_v1,
     "spincore_FID_nutation_v2": proc_FID_v1,
+    "spincore_general": proc_spincore_general_v1,
```

Comment:
processing for SC noise PSD

Reply:
again, just use a lambda function here rather than very similarly named function